<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="CalculateButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>270</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate!!</string>
    </property>
   </widget>
   <widget class="QLabel" name="FinalDisplayText">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>360</y>
      <width>431</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Imprint MT Shadow</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>190</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" rowstretch="10,10" columnstretch="0,0,0" columnminimumwidth="10,10,10">
     <item row="0" column="0">
      <widget class="QLabel" name="BillEnquiry">
       <property name="sizeIncrement">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Tw Cen MT</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Enter billed amount</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="BillAmount"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="TipEnquiry">
       <property name="sizeIncrement">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Tw Cen MT</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Enter Tip Percentage</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="TipPecentage"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="PercentageLabel">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>945</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTip_Calculator">
    <property name="title">
     <string>Tip Calculator</string>
    </property>
   </widget>
   <addaction name="menuTip_Calculator"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
